src/ctelnet.cpp

```diff
@@ -1527,15 +1527,21 @@ void cTelnet::processTelnetCommand(const std::string& command)
                 output += OPT_CHARSET;
 
                 if (!acceptedCharacterSet.isEmpty()) {
+                    QByteArray value;
                     if (acceptedCharacterSet.contains(QByteArray("ASCII"))) {
-                        setEncoding(QByteArray("ASCII"), true); // Force variants of ASCII to ASCII
+                        value = QByteArray("ASCII");
+                        setEncoding(value, true); // Force variants of ASCII to ASCII
                     } else if (acceptedCharacterSet.startsWith("ISO-")) {
-                        setEncoding(QByteArray("ISO " + acceptedCharacterSet.mid(4)), true); // Align with TEncodingTable::csmEncodings
+                        value = QByteArray("ISO " + acceptedCharacterSet.mid(4));
```

Comment:
Deliberately dropped the minus sign?

Reply:
Hmm, I don't see it - can you point it out?

Reply:
It's in the line before. 
Wasn't changed by this PR though. 
I think it will be fine

Reply:
Ah it's because Mudlet encoding name doesn't use the dash, but some servers can send it in that format.